pontoon/localizations/templates/localizations/includes/resources.html

```diff
@@ -11,13 +11,11 @@
   {{ ResourceList.header(deadline, priority) }}
 
   {% for resource in resources %}
-    {% if not loop.last %}
-      {% set main_link = url('pontoon.translate', locale.code, project.slug, resource.title) %}
-      {% set chart_link = main_link %}
-      {% set latest_activity = resource.latest_activity %}
-      {% set chart = resource.chart %}
-      {{ ResourceList.item(resource, main_link, chart_link, latest_activity, chart, deadline, priority) }}
-    {% endif %}
+    {% set main_link = url('pontoon.translate', locale.code, project.slug, resource.title) %}
+    {% set chart_link = main_link %}
```

Comment:
Why not just use `main_link`?

Reply:
Probably to make it more explicit. Should probably switch to named arguments.